### PR TITLE
Mac osx support

### PR DESCRIPTION
**Pull request by abhiramm7**
1. Updated tests for to look for .so and .dll based on platform
2. Added support for macos 
3. Updated Readme.rst

### CODE REVIEW

**Comment by bemcdonnell** [comment]
Addresses #95 #96

**Comment by bemcdonnell** [comment]
@goanpeca, Can you help with TravisCI for these changes?

**Comment by bemcdonnell** [comment]
@abhiramm7, is there a smaller open-storm logo we can reference?

**Comment by bemcdonnell** [comment]
@abhiramm7, I am going to turn travis on for pyswmm. Please update the `.travis.yml` file to the following:
```
language: python
python:
  - "2.7"
  - "3.4"
  - "3.5"
  - "3.6"
# command to install dependencies
install: "pip install -r requirements.txt"
# command to run tests
script: pytest
```

File: setup.py [hunk 1]
```diff
@@ -54,14 +54,16 @@ def get_description():
...
+                       'LICENSE.txt', 'AUTHORS']},
     package_data={'': ['lib/windows/swmm5.dll', 'LICENSE.txt', 'AUTHORS']},
     include_package_data=True,
     license="BSD2 License",
     keywords="swmm5, swmm, hydraulics, hydrology, modeling, collection system",
     classifiers=[
         "Topic :: Scientific/Engineering",
         "Topic :: Documentation :: Sphinx",
-        "Operating System :: Microsoft :: Windows",
+        "Operating System :: Microsoft :: Windows, Apple :: macOS",
```
Comment (bemcdonnell) [inline setup.py]: look up python classifiers:

https://docs.python.org/3/distutils/setupscript.html#additional-meta-data

File: setup.py [hunk 2]
```diff
@@ -54,14 +54,16 @@ def get_description():
     author_email='bemcdonnell@gmail.com',
     install_requires=REQUIREMENTS,
     packages=find_packages(exclude=['contrib', 'docs', 'tests*']),
+    package_data={'': ['lib/windows/swmm5.dll', 'lib/macos/swmm5.so' ,
```
Comment (bemcdonnell) [inline setup.py]: maybe Github is not showing this diff correct, but is package_data being overwritten?

File: pyswmm/swmm5.py [hunk 3]
```diff
@@ -501,8 +504,8 @@ def getSimUnit(self, unittype):
         >>> swmm_model.swmm_close()
         """
         value = ctypes.c_int()
-        errcode = self.SWMMlibobj.swmm_getSimulationUnit(unittype,
-                                                         ctypes.byref(value))
+        errcode = self.SWMMlibobj.swmm_getSimulationUnit(
```
Comment (bemcdonnell) [inline pyswmm/swmm5.py]: I would "un" line break these and get ciocheck running.  These should all be solved with that

Comment (abhiramm7) [inline pyswmm/swmm5.py]: Will do, i'll get it running

File: setup.py (hunk 2 above)
Comment (bemcdonnell) [inline setup.py]: @abhiramm7, it looks like `package_data` is being overwritten.  add it into the list

**Comment by bemcdonnell** [comment]
Travis file, not appveyor

**Comment by abhiramm7** [comment]
My bad !

**Comment by bemcdonnell** [comment]
One we get this working, we'll squash all these bad commits.

**Comment by bemcdonnell** [comment]
@abhiramm7 you can also wait until Gonzalo is available. He's got lots of experience with Travis

**Comment by abhiramm7** [comment]
I think i found the error with travis, the dependencies were not loaded. I'll give it an other try. @bemcdonnell

**Pull request merged**